content/index.rst

```diff
@@ -21,7 +21,6 @@ OpenFOAM training workshop
    :maxdepth: 1
    :caption: Reference
 
-   setup
```

Comment:
There is no file called `setup.rst`, so I removed it. Other changes are minor and technical

Reply:
probably it is from the previous version.